src/bin/deku_node.ml

```diff
@@ -150,6 +150,12 @@ let node folder prometheus_port =
 (* TODO: https://github.com/ocaml/ocaml/issues/11090 *)
 let () = Domain.set_name "deku-node"
 
+let () =
+  (* This is needed because Dream will initialize logs lazily *)
+  Dream.initialize_log ~enable:false ();
+  Logs.set_reporter (Json_logs_reporter.reporter Fmt.stdout);
+  Logs.set_level (Some Info)
+
```

Comment:
Is this the way to prevent Dream from formatting the output as in @anchpop's former PR?

Reply:
`Dream.initialize_log ~enable:false ();` tells Dream not to add it's own reporter and level.
```
  Logs.set_reporter (Json_logs_reporter.reporter Fmt.stdout);
  Logs.set_level (Some Info)
```
sets our custom reporter and a log level

Reply:
Ok thanks